api/AnimationEvent.json

```diff
@@ -0,0 +1,280 @@
+{
+  "api": {
+    "AnimationEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEvent",
+        "support": {
+          "webview_android": {
+            "version_added": null,
+            "prefix": "webkit"
+          },
+          "chrome": [
+            {
+              "version_added": "43"
+            },
+            {
+              "version_added": "1",
+              "prefix": "webkit"
+            }
+          ],
+          "chrome_android": {
+            "version_added": "43"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "6"
+          },
+          "firefox_android": {
+            "version_added": "6"
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": [
+            {
+              "prefix": "o",
+              "version_added": "12"
+            },
+            {
+              "version_added": "12.1"
+            },
+            {
+              "prefix": "webkit",
+              "version_added": "30"
+            }
+          ],
+          "opera_android": [
+            {
+              "prefix": "o",
+              "version_added": "12"
+            },
+            {
+              "version_added": "12.10"
+            },
+            {
+              "prefix": "webkit",
+              "version_added": "30"
+            }
+          ],
+          "safari": [
+            {
+              "prefix": "webkit",
+              "version_added": "4"
+            },
+            {
+              "version_added": "9.1"
+            }
+          ],
+          "safari_ios": {
+            "prefix": "webkit",
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "AnimationEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEvent/AnimationEvent",
+          "support": {
+            "webview_android": {
+              "version_added": false
```

Comment:
Should be `true` and prefix removed in `"43"`. Repeat for Chrome and Chrome Android.